src/Form/Type/RedirectType.php

```diff
@@ -19,12 +19,27 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
         $builder
             ->add('source', TextType::class, [
                 'label' => 'setono_sylius_redirect.form.redirect.source',
+                'required' => true,
             ])
             ->add('destination', TextType::class, [
                 'label' => 'setono_sylius_redirect.form.redirect.destination',
+                'required' => true,
             ])
             ->add('permanent', CheckboxType::class, [
                 'label' => 'setono_sylius_redirect.form.redirect.permanent',
+                'required' => false,
+            ])
+            ->add('enabled', CheckboxType::class, [
+                'label' => 'setono_sylius_redirect.form.redirect.enabled',
+                'required' => false,
+            ])
+            ->add('relative', CheckboxType::class, [
```

Comment:
Not sure we need this?

Reply:
We definitely need an enable/disable function
And the "relative" is to enable redirecting to an external site which is also a useful functionnality

Reply:
The aim here is to not store shop domain in the database when not needed. It's definitely an issue when developping on a URL and then going into production, and also when changing site domain or using multiple domains. I'm sure it's a must have feature.

Reply:
Maybe we are misunderstanding each other, but in the current implementation you input a path and an URL in the destination field. You don't need a hostname to make a redirect. See this image:

<img width="1539" alt="screen shot 2018-10-16 at 08 16 29" src="https://user-images.githubusercontent.com/2412177/46996143-e6205180-d11b-11e8-9926-9062435cd9d9.png">

Here you can see two redirects; a local and an external redirect, and they both work.

Reply:
Ow indeed, i thought you had to write the shop domain in the first redirect to make it work. My bad.
Still, I think that the input in the form is nice with the javascript I added, the user sees the real url in the end (with input labels).
The first question everyone asks when creating a redirect is : "Have I to add the first "/" or not ?" and then : "The destination must contains my shop URL or not ?"
Speaking from experience there.

S
o maybe we can leave it in the form, but unmapped, and remove it from the entity and the redirect path generation.

Reply:
Ow indeed, I forgot about the multi channel/hosts part.
I guess we need to remove this functionality because I don't see how we can handle this. Sad

Reply:
Or for a future version :
Handle redirection according to channel (or for all channels)
You might want to redirect users in one channel but not in another and routes might have the same URL.

Reply:
We might make redirect enabled on one or another channel

Reply:
Yes, but let's make that a future feature. I will make an issue for that. Will you remove the labels and js :)